Collector/ProfileClientFactory.php

```diff
@@ -6,6 +6,7 @@
 use Http\Client\HttpAsyncClient;
 use Http\Client\HttpClient;
 use Http\HttplugBundle\ClientFactory\ClientFactory;
+use Psr\Http\Client\ClientInterface;
```

Comment:
I have checked out at your branch. I run `composer update` but still this namespace is not discovered by PHPStorm.. and the package is not installed.

Could it be that exists localy on you because you have not run `composer update`?

Reply:
You need to make sure you install php-http/client-common:2.0-dev. If not you will get httplug: 1.1

Reply:
Oh, you are correct.